copyright

```diff
@@ -386,6 +386,21 @@ Copyright: Copyright Mike Koenig
 License: CC-BY-SA-3.0
 Comment: Taken from http://soundbible.com/1467-Grenade-Explosion.html
 
+Files: sounds/missile hit.wav
+Copyright: Copyright "Nbs Dark"
+License: Public Domain
+Comment: Taken from https://freesound.org/people/Nbs%20Dark/sounds/94185/
```

Comment:
The credit for the torpedo hit is missing.

Also, because of how this file is parsed, you need to use a '?' in place of spaces in the file names, i.e. "sounds/missile?hit.wav".

Reply:
My bad, the torpedo hit sound is one I've had on my computer for nearly 20 years, taken from a public domain sound website that's long since ceased to exist.